### DISCUSSION TITLE
How to import CommonJS modules in nuxt3 plugin?

### DISCUSSION THREAD

**Discussion by hamidooChe** [nuxt/nuxt#18159] [Questions]
After updating nuxt3 to the last version 3.0.0-27283410.c53b6a5, importation of commonJS modules throw this error :
`Uncaught SyntaxError: The requested module '/_nuxt/node_modules/vue3-apexcharts/dist/vue3-apexcharts.common.js?v=5486e026' does not provide an export named 'default'`

Plugin file (implementing vue3-apexchart for example which is a commonJS module) :
```
import { defineNuxtPlugin } from '#app'
import VueApexCharts from 'vue3-apexcharts'
export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(VueApexCharts)
})
```
Module transpiled in nuxt.config.ts
`transpile: ['vue3-apexcharts'],`

The problem does not exist in production mode.
It was working before the last version, maybe we need a documentation about importing CommonJS modules inside a nuxt plugin.

Thank you 🌹

**Comment by michaellindstromm** [comment]
Similar issue using supabase [https://www.npmjs.com/package/@supabase/supabase-js](https://www.npmjs.com/package/@supabase/supabase-js) server side in a plugin.
